README.md

```diff
@@ -1,58 +1,49 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+Book Store
 
 > Description the project.
 
+        An App in which a user inputs a book title and Author and it is displayed.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Javscript
+- Frameworks - React
+- Technologies used - CSS
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] OPTIONAL
Kindly endeavour to remove/update redundant demo link. This will make your ReadMe more professional.